server/views/components/new-site-notice/index.njk

```diff
@@ -1,7 +1,7 @@
 <div class="row row--yellow row--text-bar new-site-notice">
   <div class="container">
     <div class="grid">
-      <div class="{{ {shiftL: 1} | gridClasses }}">
+      <div class="{{ {} | gridClasses }}">
```

Comment:
I'm not sure whether to put the class in - or use the `gridClasses`... I quite like using this so it would change if we update the base class (moving towards a more CSS modules type approach)